feat: deprecate node v12 (#8956)
renovatebot/renovate: docs/development/bump-node-major.md

> # How to bump Renovate to next NodeJS lts release

Reviewer: ```suggestion
# How to bump Renovate to next NodeJS LTS release
```

LTS is an abbreviation, and should be capitalized.

## Add new NodeJS version

> - Add new versions via `package.json>engines>node` and `package.json>engines-next>node`

Reviewer: ```suggestion
- Add new version via `package.json>engines>node` and `package.json>engines-next>node`
```

Whoops, missed a spot.
We're using the singular form `version` everywhere else, so probably should be consistent throughout.

> - Update node at [local-development](./local-development.md)

Reviewer: ```suggestion
- Update the node version in the [local-development](./local-development.md) docs
```

> - Update node versions at Github Actions

Reviewer: ```suggestion
- Update the node versions in the Github Actions workflow files
```

> - Update the node versions in the Github Actions workflow files

Reviewer: ```suggestion
- Update the node version in the Github Actions workflow files
```

Use `version` instead of `versions` here as well... 😄

## Deprecate old NodeJS version

> - Deprecate old lts via `package.json>engines-next>node`

Reviewer: ```suggestion
- Deprecate old LTS via `package.json>engines-next>node`
```

Abbreviations should be capitalized.

> - Update node [local-development](./local-development.md)

Reviewer: ```suggestion
- Update the node version in the [local-development](./local-development.md) docs
```

Make this consistent with above suggestion.

> - Remove node versions at Github Actions

Reviewer: ```suggestion
- Remove the old node version in the Github Actions workflow files
```

## Remove old NodeJS version

- Update `package.json>engines>node`

> - Mark PR as `BREAKING`

Reviewer: Reading this, it's not clear how to `Mark PR as BREAKING`. 😕 
Do you put `BREAKING feat:` in the PR title, or do you put `BREAKING` in the commit message?

Please explain precisely how to mark the PR breaking. 😄

Author: Add label `breaking` and text `BREAKING CHANGE: ...`  🙃

Author: @HonkingGoose Can you make a suggestion now? 🙃

Reviewer: ```suggestion
- Mark PR as `BREAKING` by:
  - Adding the label `breaking` to the PR
  - Putting the text `BREAKING CHANGE:` in the PR title
```

Let's see if this renders properly. 😄

Author: `BREAKING CHANGE: <breaking cause>` comes to pr content

Reviewer: @viceice I've updated my suggestion.

> # How to bump Renovate to next NodeJS LTS release

Reviewer 2: Note for later: we should add some sample PRs to this documentation once we have some examples. It's a process that I expect we'll go through once every 6 months, so having examples always helps.

Author: maybe just link the existing merged pr's?

Reviewer 2: Yeah, but we don't have any for this new `node-next` workflow. I am happy to wait until Node v16 happens 😄
